src/Header/Header.js

```diff
@@ -1 +1,18 @@
-import React, { Component } from 'react';
\ No newline at end of file
+import React, { Component } from 'react';
+import '../Styles/Header.scss';
+import Nav from './Nav'
+
+export default function Header(props) {
+  return (
+    <header className="pageHeader">
+      <Nav />
+      <section className="banner">
+        <h1 className="coyoHeading">
+          <div className="headingText co">CO</div>
```

Comment:
I think there is likely to be a more semantic element we could use here instead of a `div`

Reply:
This div is being used purely for positioning, and making that text block level and on two separate lines within the h1, which was my more semantic choice.  I'll look into it myself, but do you have any suggestions for a cleaner alternative @thatPamIAm ?